content/en/security/cloud_security_management/setup/agentless_scanning/_index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Setting up CSM Agentless Scanning
+kind: documentation
+---
+
+<div class="alert alert-info">Agentless Scanning for Cloud Security Management is in public beta for AWS cloud environments.</div>
+
+Agentless Scanning provides visibility into vulnerabilities that exist within your AWS hosts, running containers, Lambda functions, and Amazon Machine Images (AMIs), without requiring you to install the Datadog Agent. 
+
+
+{{% csm-agentless-prereqs %}}
+
+## Deployment methods
+
+There are two recommended ways to deploy Agentless scanners in your environment, either using cross account scanning, or same account scanning.
+
+**Note**: When using Agentless Scanning, there are additional costs for running scanners in your cloud environments. To optimize on costs while being able to reliably scan every 12 hours, Datadog recommends setting up Agentless Scanning with Terraform as the default template, as this also avoids cross-region networking. 
+
+To establish estimates on scanner costs, reach out to your [Datadog Customer Success Manager.][8]
+
+{{< tabs >}}
+{{% tab "Cross account scanning" %}}
+
+With this option, Agentless scanners are deployed on a single cloud account and are distributed across multiple regions within the account. With this deployment model, Agentless scanners are granted visibility across multiple accounts without needing to perform cross-region scans, which are expensive in practice.
+
+For larger accounts with 250 or more hosts, this is the most cost-effective option since cross-region scans are avoided, and reduces friction to manage your Agentless scanners, as they are located in a single account. You can either create a dedicated account for your Agentless scanners or choose an existing one. The account where the Agentless scanners are located can also be scanned.
+
+The following diagram illustrates how Agentless scanning works when deployed in a central cloud account:
+
+
+{{< img src="/security/agentless_scanning/agentless_advanced_2.png" alt="Diagram of Agentless scanning showing the Agentless scanner is deployed in a central Cloud account" width="90%" >}}
+
+{{% /tab %}}
+{{% tab "Same account scanning" %}}
+
+With this option, a single Agentless scanner is deployed per account. Although this can incur more costs, as it requires each Agentless scanner to perform cross-region scans per account, Datadog recommends this option if you do not want to grant cross-account permissions.
```

Comment:
```suggestion
With same account scanning, a single Agentless scanner is deployed per account. Although this can incur more costs, as it requires each Agentless scanner to perform cross-region scans per account, Datadog recommends this option if you do not want to grant cross-account permissions.
```